src/components/portaldescription/OurTeamStyles.module.scss

```diff
@@ -0,0 +1,59 @@
+.teamLine_big,
+.teamLine_small {  
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  margin: 0 auto;
+}
+
+.teamLine_big {
```

Comment:
same css selectors for block and modificator:
`teamLine_big, teamMember_avatar, teamMember_imgLink`

small, big is modificators and teamMember_avatar - is block